models/src/main/java/com/foodie/FoodieBaseEntity.java

```diff
@@ -18,12 +20,15 @@ public abstract class FoodieBaseEntity {
     @Column(name = "id", nullable = false, columnDefinition = "uuid")
     private UUID id;
 
+    @JsonIgnore
```

Comment:
Does this means you don't want to ever return these fields to the user?

Reply:
Yes. It will only be returned in the core admin interface